django_mongodb_backend/schema.py

```diff
@@ -418,3 +420,61 @@ def _field_should_have_unique(self, field):
         db_type = field.db_type(self.connection)
         # The _id column is automatically unique.
         return db_type and field.unique and field.column != "_id"
+
+    def _create_collection(self, model):
+        """
+        If the model is encrypted create an encrypted collection with the
+        encrypted fields map else create a normal collection.
+        """
+
+    def _create_collection(self, model):
+        """
+        If the model is encrypted, create an encrypted collection with the
+        encrypted fields map; else, create a normal collection.
+        """
+        db = self.get_database()
+        if getattr(model, "encrypted", False):
+            client = self.connection.connection
+            options = client._options.auto_encryption_opts
+            key_vault_namespace = options._key_vault_namespace
+            kms_providers = options._kms_providers
+            codec_options = CodecOptions()
+
+            ce = ClientEncryption(kms_providers, key_vault_namespace, client, codec_options)
```

Comment:
There is some [example code](https://pymongo.readthedocs.io/en/stable/examples/encryption.html#csfle-on-demand-credentials) that uses:
```
codec_options=client.codec_options
```
which might be more appropriate (though you wonder why `codec_options` and is a required argument if options can be retrieved from the also passed `client`).